### DISCUSSION TITLE
How to react to List Actions?

### DISCUSSION THREAD

**Discussion by Brannos1970** [ant-design-blazor/ant-design-blazor#2623] [Q&A]
I am trying to figure out how to actually use and respond to List actions. I was looking at the examples and documentation but I could not see how to respond to the actions of a line item. Is there a better example that show how to do this?

**Comment by kooliokey** [comment]
You completely control the content that gets rendered into actions (it is an array of `RenderFragment`) so you  are able to have `OnClick` event handlers on those elements if you need. Something like... (Not compiled/tested, but should be about right)

```
<AntList Class="demo-loadmore-list" DataSource="@Data" ItemLayout="ListItemLayout.Horizontal" Loading="@InitLoading">
    <ChildContent Context="item">
        <ListItem Actions="Actions">
            ... other stuff for item ...
        </ListItem>
    </ChildContent>
</AntList>

@code {
    public void OnClickEdit(MouseEventArgs args){
        Console.WriteLine("CLICKED EDIT");
    }

    public RenderFragment EditButton = __builder =>
    {
        <a href="#" onclick="OnClickEdit">Edit</a>
    };

    public IEnumerable<RenderFragment> Actions = new []{
        EditButton
    }
}
```
